bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/rest/api/automowerconnect/dto/Calendar.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.automower.internal.rest.api.automowerconnect.dto;
+
+/**
+ * @author Markus Pfleger - Initial contribution
+ */
+public class Calendar {
```

Comment:
Is this class used?

Reply:
Not yet. I created the MowerData DTO according to the api specification. I always created either:
- full dto with all members
- no members, so a placeholder to add fuctionality later

The MowerData is in use and Calendar is a part of it. Calendar itself is not used yet. Of course I can remove Calendar from the MowerData until it is needed. I just thought it would be a simpler change later on to just add to the calendar class when needed.

Whatever you prefer.